### PR TITLE
PAYARA-2231 Create New Network Listener link not working

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins test please

**Comment by MeroRai** [comment]
Jenkins test please

File: appserver/admingui/web/src/main/resources/configuration/httpListenerNew.jsf [hunk 1]
```diff
@@ -38,6 +38,8 @@
     only if the new code is made subject to such option by the copyright
     holder.
 
+    Portions Copyright [2017] [Payara Foundation]
```
Comment (Pandrex247) [inline appserver/admingui/web/src/main/resources/configuration/httpListenerNew.jsf]: Should be:
`Portions Copyright [2017] [Payara Foundation and/or its affiliates]`

**Comment by MeroRai** [comment]
Jenkins test please

**Pull request merged**